components/script/dom/element.rs

```diff
@@ -203,6 +203,7 @@ pub struct Element {
     #[no_trace]
     selector_flags: Cell<ElementSelectorFlags>,
     rare_data: DomRefCell<Option<Box<ElementRareData>>>,
+    line_number: u64,
```

Comment:
I'm not sure about adding this member to every DOM element. It seems like this adds a lot of memory usage.